crates/services/sync/src/import/back_pressure_tests.rs

```diff
@@ -50,7 +50,7 @@ struct Input {
         block_stream_buffer_size: 10,
         header_batch_size: 5,
     }
-    => is less_or_equal_than Count{ headers: 10, consensus: 10, transactions: 10, executes: 1, blocks: 50 }
+    => is less_or_equal_than Count{ headers: 10, consensus: 10, transactions: 10, executes: 1, blocks: 51 }
```

Comment:
I'm wondering why this change?

Reply:
Changed to add more wait on the header verification. From my understanding it was going too fast now and so it was making too much blocks verified.